variables.tf

```diff
@@ -166,3 +166,24 @@ variable "external_bucket_region" {
   type        = string
   default     = ""
 }
+
+##########################################
+# K8s                                    #
+##########################################
+variable "resource_limits" {
```

Comment:
can we also plumb through: parquet_resource_requests, parquet_resource_limits, weave_resource_requests, weave_resource_limits?
https://github.com/wandb/terraform-kubernetes-wandb/blob/22d4ad672954be1ce9452352510b7f4fa9625129/variables.tf#L68-L120

Reply:
to be precise, we can leave the defaults to null, but just add these variables to allow passing in such variables as inputs to this module.